src/common.scss

```diff
@@ -87,71 +57,64 @@ $aqua-forest: #6aac70;
 	}
 }
 
-/* for BU Responsive theme */
-.content-container {
+/* Integrates with BU Responsive theme */
+.content-container, .site-content {
 	display: flex;
 	padding-bottom: 60px;
-}
 
-@media screen and (max-width: 600px) {
-	// Change flex-direction from row to column, for custom ordering.
-	.content-container {
+	@media screen and (max-width: 600px) {
+		// Change flex-direction for narrow screens.
 		flex-direction: column;
 	}
 
-	// Make links come 2nd in order, after the bulb container.
-	.bulb-page-links {
-		order: 2;
+	article.bulb-learning-module {
+		flex: 70%;
+		@media screen and (max-width: 600px) {
+			flex: 1 0 auto;
+		}
 	}
 
-	// Since this is a column, change flex-basis back to auto.
-	.bulb-container {
-		flex: 1 0 auto;
-	}
-}
-
-.bulb-container {
-	flex: 0 1 auto;
-	width: 80%;
-	margin-left: auto;
-	margin-right: auto;
-}
-
-.bulb-page-links {
-	flex: 1 0 200px;
-	width: 200px;
-	// margin-right: 40px;
-	float: right;
-}
-
-.bulb-page-links__title {
-	margin: 0;
-	text-align: center;
-	font-size: 20px;
-	padding: 5px 0;
-	background-color: $lightgray;
-}
-
-.bulb-page-links__title a {
-	color: $white !important;
-}
-
-.bulb-page-links li a {
-	display: block;
-	padding: 6px 20px;
-}
-
-.current_page_item {
-	background-color: $gray !important;
-}
-
-.bulb-min-list {
-	padding: 0;
-	margin: 0;
-	list-style: none;
+	.sidebar.bulb-module-sidebar {
+		flex: 30%;
+		@media screen and (max-width: 600px) {
+			flex: 1 0 auto;
+			order: 2;
+		}
+		
+		.bulb-page-links {
+			width: 100%;
+			border: 1px solid black;
+
+			&__title {
+				margin: 0;
+				text-align: center;
+				font-size: 20px;
+				padding: 5px 0;
+				background-color: $lightgray;
+	
+				a {
+					color: $white !important;
```

Comment:
`!important` statements become impossible to override if you need (which I know is the point) but you've already got enough specificity lined up that you shouldn't need it.

Reply:
To prevent needing it and also just as good practice, I would recommend establishing `hover` `focus` `active` and `visited` states for all anchors to be sure they're aren't pulling in styles from framework elements for those states. So something like
```
.a {
     color: $white;

     &:active,
     &:visited {
           color: $white;
     }
 
     &:hover,
     &:focus {
             color: $blue; // Or whatever.
     }
}
```
Especially as you're considering that this would work with Responsi or other frameworks, you'll need to have those clear overrides for all differently styled links.